motion-1/Overview.bs

```diff
@@ -97,7 +97,7 @@ ISSUE: Add more details and examples.
 
 <pre class='propdef'>
 Name: offset-path
-Value: <<url>> | <<angle>> <<size>>? && contain? | [ <<basic-shape>> | <<path()>> ] || <<geometry-box>> | none
+Value: <<url>> | ray(<<angle>> && <<size>>? && contain?) | <<basic-shape>> && <<geometry-box>>? | <<path()>> | none
```

Comment:
Shane? Tab?

My pull request is currently consistent with Shane and Tab's proposal that the basic-shape option be
"a basic shape, optionally with a geometry box, in any order"
https://github.com/w3c/fxtf-drafts/issues/42


shape-outside defines "areas". clip-path defines a "region".
Neither deals with initial position or direction.
To support a geometry box without <basic-shape>, the offset-path spec would need to define the initial position and direction.


We see in offset-path "Values have the following meanings:" that there are currently 5 options:
<angle> ...
<basic-shape>
'path()' = path(<string>)
<url>
'none'

There is no mention of geometry box without a basic-shape. People can use inset(0) <geometry-box>. Then we know
"The initial position is defined by the left top corner of the rectangle. The initial direction is 0 degree."